### ISSUE TITLE
Categorize Add-Ons

### ISSUE THREAD

**Issue by navidkashani** [navidkashani/awesome-acf#1]
for make list more useful we can categorize add-ons... some category suggest:
- UI Helper
- Other WP Plugin
- Content 
- Choice
- relational
- jquery
- layout

**Comment by Luciaisacomputer** [comment]
Yes, categories would be awesome! This list blew my mind, and having a quick and easy way to find something would be a huge plus.
